From 257839eb661d5fed3b40aecd5cb29fe183276687 Mon Sep 17 00:00:00 2001
From: bananafunction <terathree2@web.de>
Date: Thu, 9 Sep 2021 13:31:56 +0200
Subject: [PATCH] fajita: update fingerprint patch to match SPL

Signed-off-by: bananafunction <terathree2@web.de>
---
 lineage_fajita.mk | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/lineage_fajita.mk b/lineage_fajita.mk
index 9aaa73d10a9..db31eedace7 100644
--- a/lineage_fajita.mk
+++ b/lineage_fajita.mk
@@ -38,6 +38,8 @@ PRODUCT_GMS_CLIENTID_BASE := android-oneplus
 PRODUCT_BUILD_PROP_OVERRIDES += \
     TARGET_DEVICE=OnePlus6T \
     TARGET_PRODUCT=OnePlus6T \
-    PRIVATE_BUILD_DESC="OnePlus6T-user 9 PKQ1.180716.001 1812260627 release-keys"
+    PRIVATE_BUILD_DESC="redfin-user 13 TP1A.221005.002 9012097 release-keys" \
+    BUILD_NUMBER=9012097
 
-BUILD_FINGERPRINT := OnePlus/OnePlus6T/OnePlus6T:9/PKQ1.180716.001/1812260627:user/release-keys
+# Pixel 5 October fingerprint
+BUILD_FINGERPRINT := google/redfin/redfin:13/TP1A.221005.002/9012097:user/release-keys
